spec/takeaway_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'takeaway'
+
+describe Takeaway do 
+
+  let(:menu) { double :menu, dishes: { :dish_one => 1.00, :dish_two => 2.00 }, deliver_menu: :the_menu }
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)